cg/store/api/find_business_data.py

```diff
@@ -10,26 +10,41 @@
 from cg.constants.indexes import ListIndexes
 from cg.exc import CaseNotFoundError, CgError
 from cg.store.api.base import BaseHandler
-from cg.store.filters.status_analysis_filters import AnalysisFilter, apply_analysis_filter
+from cg.store.filters.status_analysis_filters import (
```

Comment:
The linter may be configured for a line length of 80 here because these lines are less than 100 characters long. I have 100 characters for line length and my linter does not break these imports.

Reply:
LOoks ok in pre-commit_
```
repos:
  - repo: https://github.com/psf/black
    rev: 23.3.0
    hooks:
      - id: black
        args: [--line-length=100]
```

and GitHub action Pass.

Reply:
maybe in your local?